tests/phpunit/tests/admin/wpListTable.php

```diff
@@ -361,4 +361,160 @@ public function data_get_views_links_doing_it_wrong() {
 			),
 		);
 	}
+
+	/**
+	 * @dataProvider data_compat_fields
+	 * @ticket 58896
+	 *
+	 * @covers WP_List_Table::__get()
+	 *
+	 * @param string $property_name Property name to get.
+	 * @param mixed $expected       Expected value.
+	 */
+	public function test_should_get_compat_fields_defined_property( $property_name, $expected ) {
+		$list_table = new WP_List_Table( array( 'plural' => '_wp_tests__get' ) );
+
+		if ( 'screen' === $property_name ) {
+			$this->assertInstanceOf( $expected, $list_table->$property_name );
+		} else {
+			$this->assertSame( $expected, $list_table->$property_name );
+		}
```

Comment:
I don't know if that's by design, but maybe this test could also utilise the `::$list_table` class property instead of creating a local variable that stores an object of the `WP_List_Table` class.
```suggestion
		if ( 'screen' === $property_name ) {
			$this->assertInstanceOf( $expected, $this->list_table->$property_name );
			return;
		} 
		$this->assertSame( $expected, $this->list_table->$property_name );
```